src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/bytecode/patch/SpoofAppVersionPatch.kt

```diff
@@ -36,11 +38,36 @@ class SpoofAppVersionPatch : BytecodePatch(
                 "revanced_spoof_app_version",
                 StringResource("revanced_spoof_app_version_title", "Spoof app version"),
                 false,
-                StringResource("revanced_spoof_app_version_summary_on", "Version spoofed to 17.30.34"),
+                StringResource("revanced_spoof_app_version_summary_on", "Version spoofed"),
                 StringResource("revanced_spoof_app_version_summary_off", "Version not spoofed"),
                 StringResource("revanced_spoof_app_version_user_dialog_message",
-                "App version will be spoofed to 17.30.34. This will give the old UI layout, but unknown side effects may occur."
-                        + " If later turned off, the old UI layout may remain until you log out or clear the app data.")
+                "App version will be spoofed to an older version of YouTube. This will change the appearance of the app, but unknown side effects may occur."
+                        + " If later turned off, the old UI may remain until you log out or clear the app data.")
+            ),
+            ListPreference(
+                "revanced_spoof_app_version_target",
+                StringResource(
+                    "revanced_spoof_app_version_target_title",
+                    "Spoof app version target"
+                ),
+                ArrayResource(
+                    "revanced_spoof_app_version_target_entries",
+                    listOf(
+                        StringResource("revanced_spoof_app_version_target_entry_1", "17.30.35 - old UI layout"),
+                        StringResource("revanced_spoof_app_version_target_entry_2", "17.01.35 - sort videos by oldest"),
+                        StringResource("revanced_spoof_app_version_target_entry_3", "16.20.35 - non material you"),
```

Comment:
What is "material you" refering to here?

```suggestion
                        StringResource("revanced_spoof_app_version_target_entry_3", "16.20.35 - Disable material you"),
```